### ISSUE TITLE
27. Juvenile dermatomyositis -> Dermatomyositis

### ISSUE THREAD

**Issue by nicolevasilevsky** [monarch-initiative/monarch-disease-ontology-RETIRED#216]
This subclass assertion makes sense to me. Should we let orphanet know?

![image](https://cloud.githubusercontent.com/assets/6722114/24210890/b4e59fbe-0ee7-11e7-8e1b-c8969d43ad9a.png)

**Comment by cmungall** [comment]
Do we have justification beyond the label?

http://www.orpha.net/consor/cgi-bin/OC_Exp.php?Expert=221

> Onset is generally in adulthood, in some cases earlier (Juvenile DM, see this term)

I say this supports a generic interpretation, which supports the subclass

Do we have a github username for an orphanet contact to cc here?

**Comment by pnrobinson** [comment]
One issue is what exactly our subclass assertions mean. Since the etiology of dematomyositis is unknown, it is hard to say if the disease is really a subclass or if it should be a sibling. I am saying this because juvenile myositis has manifestations that "adult" dermatomyositis does not, and thus it is not possible that there is a distinct etiology.

I do not think this is necessarily a mistake if MONDO does it like this, but it occurs to me that we should be as precise as possible about the edges in MONDO.

-Peter


Peter Robinson

Professor of Computational Biology

The Jackson Laboratory for Genomic Medicine

10 Discovery Drive

Farmington, CT 06032

860.837.2095 t | 860.990.3130 m

peter.robinson@jax.org<mailto:peter.robinson@jax.org>

www.jax.org

Robinson lab: https://robinsongroup.github.io/

The Jackson Laboratory: Leading the search for tomorrow's cures

________________________________

**Comment by cmungall** [comment]
Mondo is an OWL ontology so the meaning of SubClassOf is clear and strict and non negotiable.

Class definitions are always a problem however.

I think what is happening here is that the labels are misleading. I think the Orphanet generic-sounding Dermatomyositis actually refers to a disease that may indeed be etiologically distinct. They actually have "Adult Dermatomyositis" as an exact synonym. This is a common problem in disease ontologies, search this tracker for "prototypical". MonDO is OWL based and hence not prototype based.

It's OK if the adult form to have a different etiology, grouping classes are not assumed to share etiology (unless the differentia is explicitly etiology).

In ORDO there is no shared groupings between the different forms. The defs are:

 * Orphanet:221 ! Dermatomyositis [DEF: "Dermatomyositis (DM) is a type of idiopathic inflammatory myopathy characterized by evocative skin lesions and symmetrical proximal muscle weakness."] [SYN: "Adult dermatomyositis" (exact)]
 * Orphanet:398117 ! Neonatal dermatomyositis
 * Orphanet:93672 ! Juvenile dermatomyositis [DEF: "Juvenile dermatomyositis (JDM) is the early-onset form of dermatomyositis (DM, see this term), a systemic, autoimmune inflammatory muscle disorder, characterized by proximal muscle weakness, evocative skin lesion, and systemic manifestations."]

In my opinion the way DO modeled this makes sense, though they lack child definitions:

```
       is_a DOID:633 ! myositis [DEF: "A myopathy characterized by muscle inflammation."]
        is_a DOID:10223 ! dermatomyositis ***  [DEF: "A myositis that results_in inflammation located_in muscle or located_in skin where a skin rash is often seen prior to the onset of muscle weakness. The disease may result from either a viral infection or an autoimmune reaction."]
         is_a DOID:14202 ! adult dermatomyositis
         is_a DOID:14203 ! childhood type dermatomyositis
```

(I would go further and move all weasel qualifiers to the gloss, and have an OWL definition.)

I think the ORDO classes belong as subclasses of dermatomyositis.

But this prototype thing is a problem. Some people may be using Orphanet:221 as a grouping (based on the text in the web page, which leans towards generic) others as the specific adult form.

**Comment by Orphanet** [comment]
Hi there,
I'm part of the Orphanet team and I will be happy to let Annie Olry (who is in charge of the classifications) check this point.

Best regards,
Marc Hanauer


Envoyé de mon iPhone

**Comment by nicolevasilevsky** [comment]
@annieolry
